omnibus/config/software/openssl-fips-provider.rb

```diff
@@ -0,0 +1,35 @@
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https:#www.datadoghq.com/).
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Isn't it `https://datadoghq.com` instead? (same for the other ruby files)

Reply:
Idk tbh, I just copied this from other files (26 occurences so far). I've changed it for files I created but I'm not confident to do it for others as I don't know the reason why it was written this way.

Reply:
5f7018a3c8ed752b26c1a27ab728ee75dd27dee0